cleanup_delete.go

```diff
@@ -78,8 +81,8 @@ func (dq *deleteQueue) deleteFromPending() {
 	defer dq.Unlock()
 
 	n := len(dq.entries)
-	if n > awsBatchSizeLimit {
-		n = awsBatchSizeLimit
```

Comment:
I believe the previous behaviour was correct and unrelated to the change you want to make.  In this case, we're capping the number of items sent via the `DeleteMessageBatch` call inside the `deleteBatch` method: https://github.com/Wattpad/sqsconsumer/blob/d2be49ee1dbf7f2fcd0fafb65d34c72524258be4/cleanup_delete.go#L52-L61

AWS allows a maximum of 10 messages per `DeleteMessageBatch` call, so our code splits up the batches accordingly.  I think the change you're trying to make with regards to the number of Go routines does not affect the  "batch deleter" code.

Reply:
I agree. The previous code used one constant for: max messages to delete, max messages to receive, and the number of job workers. These three things are unrelated. So, I had to create a separate const for the job workers, and another one for the number of messages (for both receive and delete). 

How about creating a new const and we call it `DeleteMessagesBatchSizeLimit` so it is specific to the `DeleteMessageBatch`?

Reply:
That sounds good to me, makes it more clear for our future selves.  :)